### PR TITLE
feat: implement systray

### PR DESCRIPTION
**Pull request by Misieq01**
Description
---

Implement systray icon with informations about cpu, gpu and estimated earnings 

Motivation and Context
---

How Has This Been Tested?
---

Manually

What process can a PR reviewer use to test or verify this change?
---

Open app and check system tray

Breaking Changes
---

- [x] None
- [ ] Requires data directory on base node to be deleted
- [ ] Requires hard fork
- [ ] Other - Please specify

### CODE REVIEW

**Comment by Misieq01** [comment]
Windows: 
image

From what I understand there is limit of 64 characters for tooltip on windows so i had to make it shorter

**Comment by Misieq01** [comment]
Linux: 
image

Tooltip is not supported there so instead it shows parameters on click

**Comment by Misieq01** [comment]
macOS: 
image

File: src-tauri/src/systemtray_manager.rs [hunk 1]
```diff
@@ -0,0 +1,234 @@
+use log::{error, info};
+use std::sync::LazyLock;
+use tauri::{AppHandle, CustomMenuItem, SystemTray, SystemTrayMenu, SystemTrayMenuItem};
+
+use crate::hardware_monitor::HardwareStatus;
+
+const LOG_TARGET: &str = "tari::universe::notification_manager";
```
Comment (stringhandler) [inline src-tauri/src/systemtray_manager.rs:7]:
```suggestion
const LOG_TARGET: &str = "tari::universe::systemtray_manager";
```

**Pull request merged**